src/patternfly/components/Button/examples/button-types-example.hbs

```diff
@@ -11,8 +11,9 @@
     Danger
 {{/button}}
 {{#> button button--modifier="pf-m-link"}}
-    {{> button-icon button-icon--type="plus-circle" button-icon--modifier="pf-c-button__icon"}}
+    {{> button-icon button-icon--type="times-circle"}}
```

Comment:
Looks like there isn't enough space between the icon and the text in this button. Compare it to the pf-m-link examples in the "button states" section to see the proper spacing. Then check to see if you can find any other instances of this, too.

<img width="618" alt="screen shot 2019-01-04 at 9 18 27 am" src="https://user-images.githubusercontent.com/35148959/50695273-d4a36300-1001-11e9-9a56-98d151c4320c.png">

Reply:
Hey @mcoker, thanks for pointing out spacing. It's actually the wrong button icon. I placed it with the correct one `plus-circle`, not `times-circle`.